### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by Dqrshan**
Improved most of the code.

- **Discord: [Nin#1111](https://discord.com/users/838620835282812969)**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Custom
 
 # Remove comment for the line below before you push your code to GitHub!
-# .env
+.env
```
Comment (Toastyy1) [inline .gitignore]: Don't delete the comment, otherwise it wouldn't upload my example `.env` file

File: commands/test/sampleCmd.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
```
Comment (Toastyy1) [inline commands/test/sampleCmd.js]: The function shouldn't be `async` and why did you delete all the example attributes?

File: contextCmds/test/msgCmd.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    name: "content",
+    type: 3, // 2 is for "USER" | 3 is for "MESSAGE" (EXECUTED BY RIGHT CLICKING EITHER)
```
Comment (Toastyy1) [inline contextCmds/test/msgCmd.js]: You have to explain this to me

File: contextCmds/test/userCmd.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    name: "avatar",
+    type: 2, // 2 is for "USER" | 3 is for "MESSAGE" (EXECUTED BY RIGHT CLICKING EITHER)
```
Comment (Toastyy1) [inline contextCmds/test/userCmd.js]: You have to explain this to me too

File: events/guild/messageCreate.js [hunk 5]
```diff
@@ -2,29 +2,25 @@
...
 const validatePermissions = (permissions) => {
-  const { Permissions } = require("discord.js");
-  const validPermissions = Object.keys(Permissions.FLAGS);
-
-  for (const permission of permissions) {
-    if (!validPermissions.includes(permission)) {
-      throw new Error(`Unknown permission node "${permission}"`);
-    }
-  }
+	const { Permissions } = require("discord.js");
```
Comment (Toastyy1) [inline events/guild/messageCreate.js:5]: Too many spaces / tabs

File: index.js [hunk 6]
```diff
@@ -25,9 +25,12 @@ const client = new Client({
 });
 
 client.commands = new Collection();
+client.aliases = new Collection();
```
Comment (Toastyy1) [inline index.js]: We don't need an extra collection for aliases. I already found an easy way to use aliases, I'll add it as soon as possible

**Review by Toastyy1 — CHANGES_REQUESTED** [review]
I've added some comments, have a look at them

**Comment by Toastyy1** [comment]
Ah and no emojis in commits pls ^^

File: .gitignore (hunk 1 above)
Comment (Dqrshan) [reply .gitignore]: Done

File: commands/test/sampleCmd.js (hunk 2 above)
Comment (Dqrshan) [reply commands/test/sampleCmd.js]: welp my bad, I fixed that too

File: contextCmds/test/msgCmd.js (hunk 3 above)
Comment (Dqrshan) [reply contextCmds/test/msgCmd.js]: It's a context menu command (right clicking any message -> apps -> command)

File: contextCmds/test/userCmd.js (hunk 4 above)
Comment (Dqrshan) [reply contextCmds/test/userCmd.js]: It's a context menu command (right clicking any user-> apps -> command)

File: events/guild/messageCreate.js (hunk 5 above)
Comment (Dqrshan) [reply events/guild/messageCreate.js:5]: alright

File: index.js (hunk 6 above)
Comment (Dqrshan) [reply index.js]: okay i'll just remove the aliases part for now

**Review by Toastyy1 — APPROVED** [review]

**Review by Toastyy1 — APPROVED** [review]
